CanvasPlusPlayground/PeopleManager.swift

```diff
@@ -111,7 +114,11 @@ class PeopleManager {
                         return
                     }
                     
-                    guard let enrollments: [Enrollment] = try? await CanvasService.shared.syncWithAPI(request) else {
+                    guard let enrollments: [Enrollment] = try? await CanvasService.shared.loadAndSync(request, onCacheReceive: { cached in
+                        guard let cached else { return }
+
+                        processEnrollments(cached)
+                    }) else {
```

Comment:
idea:create local var `isVerified`, implement the `onBatch` closure with processEnrollments. then we set an `isVerified = true` in `processEnrollments` so we only processEnrollments once

Reply:
Solved